### DISCUSSION TITLE
docs/genuine-oj/backend

### DISCUSSION THREAD

**Discussion by giscus** [yxzlwz/blog#8] [Announcements]
# docs/genuine-oj/backend

环境准备 本文默认的系统环境为 Ubuntu 22.04，其它系统环境请自行寻找“环境准备”部分的解决方案。 apt 下载速度过慢可参考下面的网页进行换源。 ubuntu 镜像站使用帮助 清华大学开源软件镜像站 | Tsinghua Open Source Mirror (https://mirrors.tuna.tsinghua.edu.cn/hel...

https://blog.yixiangzhilv.com/docs/genuine-oj/backend.html

**Comment by hachinoone** [comment]
你好，在提交评测的时候celery返回错误：
raised unexpected: ProgrammingError('relation "oj_submission_submission" does not exist\nLINE 1: ... "oj_submission_submission"."allow_download" FROM "oj_submis...\n
psycopg2.errors.UndefinedTable: relation "oj_submission_submission" does not exist
请问如何解决？

**Reply by yxzlwz** [reply]
从这个信息来看你在运行celery的时候可能忘记配置环境变量了，麻烦检查下
